english/fork-jam/not_dreaming.md

```diff
@@ -8,4 +8,6 @@ Do you:
 
 [Drink from it?](./choco-chalice/choco-chalice.md)
 
+[Smash the altar](../altar-smash/altar-smash.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.